components/gallery/GalleryItemButton/GalleryItemMoreActionBtn.vue

```diff
@@ -151,4 +156,11 @@ const unlist = () => {
     errorMessage: $i18n.t('transaction.unlist.error') as string,
   })
 }
+
+const refreshMetadata = async () => {
+  if (props.collectionId && props.nftSn) {
+    await refreshOdaTokenMetadata(urlPrefix.value, props.collectionId!, props.nftSn!)
```

Comment:
maybe show a toast `refreshing...`, atm it looks like the click did nothing untll it refreshes

Reply:
added